WordPress/Classes/ViewRelated/Blog/Site Management/StartOverViewController.swift

```diff
@@ -15,7 +15,15 @@ open class StartOverViewController: UITableViewController, MFMailComposeViewCont
 
     @objc let headerView: TableViewHeaderDetailView = {
         let header = NSLocalizedString("Let Us Help", comment: "Heading for instructions on Start Over settings page")
-        let detail = NSLocalizedString("If you want a site but don't want any of the posts and pages you have now, our support team can delete your posts, pages, media, and comments for you.\n\nThis will keep your site and URL active, but give you a fresh start on your content creation. Just contact us to have your current content cleared out.", comment: "Detail for instructions on Start Over settings page")
+        let detail1 = NSLocalizedString("If you want a site but do not want any of the posts and pages you have now, " +
```

Comment:
I'm totally lurking here, but I have one request. Can you add a comment explaining why this is split? So the next person who comes along doesn't say "oh this can be one string" and jam it all together again.

Reply:
good point! I'll do that.

Reply:
Took the liberty to update the comment, as the 255-chars restriction is only on the key, not the translations.

Since it happens that on iOS we use the English copy as the key, then yes we will need to split that key which ends up splitting the English copy. But the translations for that key can be longer.